regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -72,6 +96,10 @@ var CommentReviewView = Backbone.View.extend({
     this.render();
   },
 
+  toggleSubmit: function() {
+    $('.submit-button').prop('disabled', function(i, v) { return !v; });
```

Comment:
Yep, I've been handling logic in templates with gross backbone template js, like `<% if (true) { %>something<% } %>`.